plugins/talk-plugin-rich-text/client/components/rte/RTE.css

```diff
@@ -1,5 +1,5 @@
 .contentEditable {
-  background: #fff;
+  background: transparent;
```

Comment:
The above suggestion would render this change unnecessary. Also this change would mean that the background color of the parent container will shine through the RTE which might not be wanted (e.g. if it's not white)

Reply:
Agreed it's unnecessary since the above change does actually solve the issue.